net-misc/ipv6calc/ipv6calc-4.3.2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit flag-o-matic toolchain-funcs
+
+DESCRIPTION="IPv6 address calculator"
+HOMEPAGE="https://www.deepspace6.net/projects/ipv6calc.html"
+SRC_URI="https://github.com/pbiering/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="cgi geoip +openssl"
+
+RDEPEND="
+	cgi? (
+		dev-perl/HTML-Parser
+		dev-perl/URI
+		www-servers/apache
+	)
+	geoip? (
+		dev-libs/geoip
+		dev-libs/libmaxminddb:=
+	)
+	openssl? ( >=dev-libs/openssl-3.0.0:= )
+	!openssl? ( app-crypt/libmd )
+"
+DEPEND="${RDEPEND}"
+
+PATCHES=(
+	# https://github.com/pbiering/ipv6calc/pull/49
+	"${FILESDIR}"/${P}-fix_directcall_ar.patch
+	"${FILESDIR}"/${P}-ldconfig_musl.patch
+)
+
+DOCS=( ChangeLog CREDITS README README.MaxMindDB README.GeoIP2 TODO USAGE )
+HTML_DOCS=( doc/ipv6calc.html )
+
+src_configure() {
+	# something is broken with clang. to investigate.
+	# see https://github.com/pbiering/ipv6calc/issues/45
+	tc-is-clang && append-ldflags $(no-as-needed) && filter-lto
```

Comment:
Maybe ancient libtool again. Does elibtoolize (or eautoreconf) owrk? I won't block on merging there though, as the package really needs this love and has been broken for some time. Thank you for this!

Reply:
nor autogen.sh or eautoreconf don't.